src/locale/locales/es/messages.po

```diff
@@ -313,23 +313,23 @@ msgstr ""
 
 #: src/screens/Profile/Header/Handle.tsx:53
 msgid "⚠Invalid Handle"
-msgstr "⚠Nombre de usuario inválido"
+msgstr "⚠Manejo inválido"
```

Comment:
```suggestion
msgstr "⚠Nombre de usuario inválido"
```

"Manejo" makes no sense here, as it refers to "handling". "Nombre de usuario" is a fitting translation for "handle"